src/documents/tests/test_consumer.py

```diff
@@ -445,6 +446,16 @@ def testOverrideTags(self):
         self.assertIn(t3, document.tags.all())
         self._assert_first_last_send_progress()
 
+    def testOverrideOwner(self):
```

Comment:
Python conventions, per PEP8, are to use `snake_case`, not `camelCase`.  Since this is new, lets use that